content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -90,17 +90,16 @@ And you will see output similar to:
 Name:         frontend
 Namespace:    default
 Selector:     tier=frontend
-Labels:       app=guestbook
+Labels:       app=nginx
               tier=frontend
-Annotations:  kubectl.kubernetes.io/last-applied-configuration:
-                {"apiVersion":"apps/v1","kind":"ReplicaSet","metadata":{"annotations":{},"labels":{"app":"guestbook","tier":"frontend"},"name":"frontend",...
+Annotations:  <none>
 Replicas:     3 current / 3 desired
 Pods Status:  3 Running / 0 Waiting / 0 Succeeded / 0 Failed
 Pod Template:
   Labels:  tier=frontend
   Containers:
-   php-redis:
-    Image:        gcr.io/google_samples/gb-frontend:v3
```

Comment:
Check PR #44290
It addressed a similar issue on another page.
The image used is `us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5`.

Using this image will save you the changes needed in the rest of the file due to the use of nginx.